notebooks/240-named-entity-recognition/240-named-entity-recognition.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ! pip install --upgrade diffusers
</pre><p>please add -q for reduce logs during installation</p><p> </p><p>optimum rely to specific openvino and nncf versions, which can be different from notebooks dependencies (prevent opportunity to check new versions until optimum is not upgraded), I recommend to use separated installation of optimum-intel package and openvino-dev and nncf</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
done